probe-rs/examples/multidrop_raw.rs

```diff
@@ -13,7 +13,7 @@ fn main() -> Result<()> {
 
     let probe_lister = Lister::new();
 
-    let probes = probe_lister.list_all();
+    let probes = async_io::block_on(probe_lister.list_all());
```

Comment:
Maybe it would be better to use `#[tokio::main]` in examples, too.

Reply:
Good point. Will do.

Reply:
If we do this we would have to pull the whole tokio into the main lib. I'd try and avoid that for now. I could pull in pollster though for something lighter. But then we mix and match, so maybe better tokio. I am undecided tbh.

Reply:
You can also do `fn main() { block_on(async { everything }) }` I guess